cloud/openstack/os_image_facts.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
+
+# Copyright (c) 2014 Hewlett-Packard Development Company, L.P.
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+try:
+    import shade
+    from shade import meta
+    HAS_SHADE = True
+except ImportError:
+    HAS_SHADE = False
+
+DOCUMENTATION = '''
+---
+module: os_image_facts
+short_description: Retrieve facts about a compute instance
+version_added: "2.0"
+author: "Davide Agnello (@dagnello)"
+description:
+    - Retrieve facts about a image instance from OpenStack.
```

Comment:
`instance` is an overloaded term. I would use a different term, or just `Retrieve facts regarding an image within OpenStack`